conf/app.routes

```diff
@@ -1,13 +1,15 @@
 # microservice specific routes
 
-GET     /pension-charges/nino/:nino/taxYear/:taxYear              controllers.PensionChargesController.getPensionCharges(nino: String, taxYear: Int)
+GET     /pension-charges/nino/:nino/taxYear/:taxYear                          controllers.PensionChargesController.getPensionCharges(nino: String, taxYear: Int)
 
-DELETE  /pension-charges/nino/:nino/taxYear/:taxYear              controllers.PensionChargesController.deletePensionCharges(nino: String, taxYear: Int)
+DELETE  /pension-charges/nino/:nino/taxYear/:taxYear                          controllers.PensionChargesController.deletePensionCharges(nino: String, taxYear: Int)
 
-PUT     /pension-charges/nino/:nino/taxYear/:taxYear              controllers.PensionChargesController.createUpdatePensionCharges(nino:String, taxYear: Int)
+PUT     /pension-charges/nino/:nino/taxYear/:taxYear                          controllers.PensionChargesController.createUpdatePensionCharges(nino:String, taxYear: Int)
 
-GET     /pension-reliefs/nino/:nino/taxYear/:taxYear              controllers.PensionReliefsController.getPensionReliefs(nino: String, taxYear: Int)
+GET     /pension-reliefs/nino/:nino/taxYear/:taxYear                          controllers.PensionReliefsController.getPensionReliefs(nino: String, taxYear: Int)
 
-PUT     /pension-reliefs/nino/:nino/taxYear/:taxYear              controllers.PensionReliefsController.createOrAmendPensionReliefs(nino: String, taxYear: Int)
+PUT     /pension-reliefs/nino/:nino/taxYear/:taxYear                          controllers.PensionReliefsController.createOrAmendPensionReliefs(nino: String, taxYear: Int)
 
-DELETE  /pension-reliefs/nino/:nino/taxYear/:taxYear              controllers.PensionReliefsController.deletePensionReliefs(nino: String, taxYear: Int)
+DELETE  /pension-reliefs/nino/:nino/taxYear/:taxYear                          controllers.PensionReliefsController.deletePensionReliefs(nino: String, taxYear: Int)
+
+DELETE  /state-benefits/nino/:nino/taxYear/:taxYear/benefitId/:benefitId      controllers.StateBenefitsController.deleteStateBenefits(nino: String, taxYear: Int, benefitId: String)
```

Comment:
this URL is different to the one in the DES spec

Reply:
Yes this is the URL to this service and not the URL to call DES

The matching URL in the DES spec you will find in the StateBenefitsConnector where it calls DES with that URL
(line 49)